src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,169 @@
+import { Loader } from '../Loader';
+import React, { useEffect, useState } from 'react';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import cs from 'classnames';
+import { Link, useLocation, useNavigate } from 'react-router-dom';
+
+enum Error {
+  NoError = 'No Error',
+  NoPeople = 'No People',
+  SomethingWentWrong = 'Something Went Wrong',
+}
+
+const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState<Error>(Error.NoError);
+  const [loading, setLoading] = useState<boolean>(false);
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then(data => {
+        if (!data.length) {
+          setError(Error.NoPeople);
+        } else {
+          setError(Error.NoError);
+        }
```

Comment:
```suggestion
          setError(Error.NoPeople);
```
why do you need to setup NoError state?
it is default state and no need to setup again.